uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -366,32 +366,47 @@ Control {
                     }
                 }
             }
+            ToolSeparator {
+                Layout.fillWidth: true
+                Layout.columnSpan: 3
+                orientation: Qt.Horizontal
+            }
```

Comment:
using toolseparator on a single row, to create a separator between components.  Could use a `Rectangle` height 1, but each style would have hard-coded the color I set to it. @anmacdonald is that a good idea?

Reply:
I don't fully understand the problem here?

Reply:
Not really a problem, just checking that using a `ToolSeparator` in a ListView to separate the component seems a good solution. It looks like this in default Qt style and similar in calcite:
![image](https://user-images.githubusercontent.com/91129623/153001660-1b1eeb30-8266-4c33-ba7d-9499afc625b2.png)

Reply:
Oh I see, no that makes sense to me!